cmd/mt-split-metrics-by-ttl/main.go

```diff
@@ -32,7 +32,8 @@ var (
 	cassandraUsername = flag.String("cassandra-username", "cassandra", "username for authentication")
 	cassandraPassword = flag.String("cassandra-password", "cassandra", "password for authentication")
 
-	windowFactor = flag.Int("window-factor", 20, "the window factor be used when creating the metric table schema")
+	windowFactor             = flag.Int("window-factor", 20, "the window factor be used when creating the metric table schema")
+	cassandraOmitReadTimeout = flag.Int("cassandra-omit-read-timeout", 60, "if a read is older than this, it will directly be omitted without executing")
```

Comment:
actually, same for some other options like cassandraReadConcurrency. can just hardcode such values and comment why